<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdd</class>
 <widget class="QDialog" name="DialogAdd">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Sync</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo Sync</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalva">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSource">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDestinazione">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSource">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sorgente...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDestinazione">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Destinazione...</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>btnSource</tabstop>
  <tabstop>btnDestinazione</tabstop>
  <tabstop>btnSalva</tabstop>
  <tabstop>txtSource</tabstop>
  <tabstop>txtDestinazione</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
